packages/pn-pa-webapp/src/components/NewNotification/__test__/PreliminaryInformationsContent.test.tsx

```diff
@@ -0,0 +1,131 @@
+
+import { PhysicalCommunicationType } from '@pagopa-pn/pn-commons';
+import {
+  fireEvent,
+  getById,
+  getByText,
+  render,
+  testFormElements,
+} from '@pagopa-pn/pn-commons/src/test-utils';
+
+import { NewNotificationLangOther, PaymentModel } from '../../../models/NewNotification';
+import { PreliminaryInformationsPayload } from '../../../redux/newNotification/types';
+import PreliminaryInformationsContent from '../PreliminaryInformationsContent';
+
+// mock imports
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: { language: 'it' },
+  }),
+}));
+
+describe('PreliminaryInformationsContent', () => {
+  const initialValues: PreliminaryInformationsPayload = {
+    lang: 'it',
+    additionalLang: '',
+    paymentMode: PaymentModel.NOTHING,
+    paProtocolNumber: '',
+    subject: '',
+    physicalCommunicationType: PhysicalCommunicationType.AR_REGISTERED_LETTER,
+    taxonomyCode: '',
+  };
+
+  const mockLanguages = {
+    it: 'Italiano',
+    de: 'Tedesco',
+  };
+
+  const mockOnChangeTouched = jest.fn();
+  const mockSubjectHelperText = 'Helper text for subject';
+
+  it('renders the component', () => {
+    const { container } = render(
+      <Formik initialValues={initialValues} onSubmit={() => {}}>
+        {(formik) => (
+          <PreliminaryInformationsContent
+            formik={formik}
+            languages={mockLanguages}
+            onChangeTouched={mockOnChangeTouched}
+            subjectHelperText={mockSubjectHelperText}
+          />
+        )}
+      </Formik>
+    );
+
+    testFormElements(container, 'subject', 'subject*');
+    testFormElements(container, 'abstract', 'abstract');
+  });
+
+  it('calls onChangeTouched when subject field changes', () => {
+    const { container } = render(
+      <Formik initialValues={initialValues} onSubmit={() => {}}>
+        {(formik) => (
+          <PreliminaryInformationsContent
+            formik={formik}
+            languages={mockLanguages}
+            onChangeTouched={mockOnChangeTouched}
+            subjectHelperText={mockSubjectHelperText}
+          />
+        )}
+      </Formik>
+    );
+
+    const subjectInput = getById(container, 'subject');
+    fireEvent.change(subjectInput, { target: { value: 'New Subject' } });
+
+    expect(mockOnChangeTouched).toHaveBeenCalled();
+  });
+
+  it('displays helper text when subject field is focused', () => {
+    const { container } = render(
+      <Formik initialValues={initialValues} onSubmit={() => {}}>
+        {(formik) => (
+          <PreliminaryInformationsContent
+            formik={formik}
+            languages={mockLanguages}
+            onChangeTouched={mockOnChangeTouched}
+            subjectHelperText={mockSubjectHelperText}
+          />
+        )}
+      </Formik>
+    );
+
+    const subjectInput = getById(container, 'subject');
+    fireEvent.focus(subjectInput);
+
+    expect(getByText(container, mockSubjectHelperText)).toBeInTheDocument();
```

Comment:
Don't import getByText but use that from the return of render function. Do it in all tests